bgpd/bgp_vty.c

```diff
@@ -10655,7 +10655,7 @@ static int bgp_show_neighbor(struct vty *vty, struct bgp *bgp,
 		if (use_json)
 			json_object_boolean_true_add(json, "bgpNoSuchNeighbor");
 		else
-			vty_out(vty, "%% No such neighbor\n");
+			vty_out(vty, "%% No such neighbor in this view/vrf\n");
```

Comment:
This is much better :) I see we still have this `No such neighbor` error message in a few other spots in bgpd (e.g. `show_ip_bgp_neighbor_routes_cmd`). Would it be possible to update them as well?

Reply:
+1. agree with Renato

Reply:
Absolutely.  Thanks for pointing it out.